backend/pennclubs/settings/development.py

```diff
@@ -25,4 +25,9 @@
 CHANNEL_LAYERS = {"default": {"BACKEND": "channels.layers.InMemoryChannelLayer"}}
 
 # Caching settings
-CACHES = {"default": {"BACKEND": "django.core.cache.backends.locmem.LocMemCache"}}
+CACHES = {
+    "default": {
+        "BACKEND": "django.core.cache.backends.locmem.LocMemCache",
+        "TIMEOUT": 60 * 60,
```

Comment:
Can we revert the timeout change? Rather keep the API consistent

Reply:
Yup, makes sense.